manifests/leader-worker-set.clusterserviceversion.yaml

```diff
@@ -0,0 +1,456 @@
+      mediatype: image/png
+  install:
+    spec:
+      clusterPermissions:
```

Comment:
Does this have to be updated with each change in the operand or in the operator? It might be better to update this via our script as well.

Reply:
Actually, as far as I understand it is expected to run `operator-sdk generate bundle --input-dir deploy --version 0.1.0 --channels=tech-preview --default-channel=tech-preview --package leader-worker-set --output-dir=.` each time and this will be populated based on whatever defined under `deploy` directory

Reply:
I see, that sounds great.